test/GDPR_consent_categories.spec.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const GDPR_CONSENT_CATEGORIES = require('../GDPR_consent_categories');
+const GDPR_CONSENTS = require('../GDPR_consents');
+
+describe('consent_categories', function() {
```

Comment:
this description does not match the filename

Reply:
fixed